iam_migrations/freeipa/src/main.rs

```diff
@@ -952,10 +958,14 @@ fn main() {
         .init();
 
     // Startup sanity checks.
+    #[cfg(target_family = "unix")]
```

Comment:
Similar this for the junk draw for "check current user permission is not root/admin"